test/integration/test/stream.js

```diff
@@ -0,0 +1,73 @@
+const {AudioConfig} = require('../../../dist/src/models/streaming/AudioConfig');
+const {RevAiStreamingClient} = require('../../../dist/src/streaming-client');
+const configHelper = require('../src/config-helper');
+const fs = require('fs');
+const assert = require('assert');
```

Comment:
Is this a built in Node package? If not you'll need to add it to the dev requirements